Zend/zend_builtin_functions.stub.php

```diff
@@ -2,6 +2,7 @@
 
 /** @generate-class-entries */
 
+/** @undocumentable */
```

Comment:
I'm not sure about that. While the documentation for `stdClass` would only be terse, it might nonetheless sense having it (if only to help users finding it).